conmon/conmon.c

```diff
@@ -1321,6 +1321,18 @@ int main(int argc, char *argv[])
 		if (dup2(slavefd_stderr, STDERR_FILENO) < 0)
 			pexit("Failed to dup over stderr");
 
+		/* If LISTEN_PID env is set, we need to modify to set
```

Comment:
nit of a nit.  suggest removing "to modify"  and maybe add 'pid' at the end?

Reply:
fixed.